pkg/clusters/addons/kong/vars.go

```diff
@@ -53,4 +53,8 @@ const (
 	// DefaultProxyNodePort indicates the default NodePort that will be used for
 	// the proxy when applicable.
 	DefaultProxyNodePort = 30080
+
+	// DefaultAdminNodePort indicates the default NodePort that will be used for
+	// the admin API when applicable.
+	DefaultAdminNodePort = 32080
```

Comment:
We added this, but I'm not seeing where we're using this? Is this just for caller reference, or am I missing something?

Reply:
Indeed we're not using this 🤔 Has to be an artifact of trying to be consistent with defining that as it's done for proxy port. I'm open to both leaving or removing this.

Reply:
I think it's fine, was just making sure we weren't _mistakenly_ not using it.